### DISCUSSION TITLE
This is actually awesome

### DISCUSSION THREAD

**Discussion by sqlongithub** [PXAV/kelp#52] [General]
I am gonna use this since for some reason sound names get changed so often.

**Comment by PXAV** [comment]
Thank you for appreciating the repo! I also think this will help many developers. Feel free to use and test it, but always keep in mind this is not yet fully released and the API code is still subject to change.

**Comment by toydotgame** [comment]
This will certainly help when I expand my plugins' goals. Currently they're utility-based simple plugins, and I use Spigot 1.6.4 to compile - which works even up to 1.16.5 (current version, as of writing).

**Reply by PXAV** [reply]
Wow, that's impressive - some things didn't really change since 1.6. But as you already said, this only applies for very basic operations with the bukkit api and things start becoming more difficult if you want to work with packets, sounds, etc.

**Reply by toydotgame** [reply]
Yeah, my Spigot development is weird.
My main plugin, [Thisway](https://github.com/toydotgame/Thisway), uses only `org.bukkit.*` imports. _Then_, it's compiled with Spigot's **_server_** JAR. It really is a miracle that it works on 1.16!
I can thank the fact that when there's a bug in Spigot 1.6, and Minecraft 1.7 is released (_and_ the bug from 1.6 also exists in 1.7 Spigot builds) in the meantime - the bug gets fixed in 1.7 and 1.6 gets no fix. Thanks to just how solid and linear Spigot (and upsteam at Bukkit, also) is developed, forwards-compatability is made very easy. But also, I'm quite lucky that it can work as well. :P
